backend/src/main/java/com/htecgroup/skynest/model/entity/ObjectEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.htecgroup.skynest.model.entity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Entity(name = "object")
+@Inheritance(strategy = InheritanceType.JOINED)
+public class ObjectEntity implements Serializable {
+
+  private static final long serialVersionUID = 145843022185009528L;
+
+  @Id @GeneratedValue private UUID id;
+
+  @Column(insertable = false)
+  private LocalDateTime createdOn;
```

Comment:
You can use  @CreatedDate and @LastModifiedDate for this audition tables, just you need to allow EntityListeners in your project.

The other option is to use 
    @PrePersist
    @PreUpdate
    @PreRemove